src/test/resources/agents/bipush.yaml

```diff
@@ -0,0 +1,15 @@
+opcodes:
+  - BIPUSH 10
+  - BIPUSH 20
+  - IADD
+agents:
```

Comment:
@volodya-lombrozo try to change the order of the agents. Maybe not in this test, but in a new one.